handlers.go

```diff
@@ -38,13 +40,16 @@ func (sf *Sunfish) AddFile(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 		w.WriteHeader(422) // unprocessable entity
 		if err := json.NewEncoder(w).Encode(err); err != nil {
-			panic(err)
+			sf.logger.Println("ERROR: Could not encode json response.")
+			return
 		}
+		sf.logger.Println("INFO: Could not process the uploaded Siafile.")
```

Comment:
might be better not to create a log entry for something like this, and instead just provide a detailed error message to the uploader. I'm just wary of someone being able to intentionally clutter the logs.